src/main/java/games/strategy/engine/data/properties/GameProperties.java

```diff
@@ -129,6 +135,19 @@ public void addEditableProperty(final IEditableProperty property) {
     return properties;
   }
 
+  public void addPlayerProperty(final IEditableProperty property) {
+    playerProperties.put(property.getName(), property);
+  }
+
+  /**
+   * Return list of player properties in the order they were added.
+   *
+   * @return a list of IEditableProperty
+   */
```

Comment:
Yeah, ended up going a different direction so need to remove this comment. I'll also switch to HashMap as I think you'll always want to get player properties by the specific player not order added.